test/compat/GetFunctionBody.phpt

```diff
@@ -0,0 +1,76 @@
+--TEST--
+Retrieve the AST or code directly from a function body
+--FILE--
+<?php
+require_once __DIR__ . '/../../vendor/autoload.php';
+
+$source = <<<'EOF'
+<?php
+
+function adder(int $a, int $b)
+{
+    return $a + $b;
+}
+EOF;
+
+
+$reflector = new \BetterReflection\Reflector\FunctionReflector(
+    new BetterReflection\SourceLocator\Type\StringSourceLocator($source)
+);
+
+$functionInfo = $reflector->reflect('adder');
+var_dump($functionInfo->getName());
+
+var_dump($functionInfo->getBodyAst());
+var_dump($functionInfo->getBodyCode());
+
+?>
+--EXPECTF--
+string(5) "adder"
+array(1) {
```

Comment:
This test seems too fragile, and may break if PHP-Parser changes internal properties representation. `getBodyCode` is sufficient.